README.md

```diff
@@ -10,23 +10,31 @@ Create a fork of the original project [GoogleChromeLabs/sample-pie-shop](https:/
 
 ### Install dependencies
 
-From the top level project directory run the following:
+Change to the top level project directory run the following:
 
 ```sh
 cd sample-pie-shop
-npm install
+npm ci
 ```
 
+*Note:* This will install the dependencies as per the `package-lock.json` whereas `npm install` will update them. If you need to update the dependencies, submit the updated `package-lock.json` as a new PR.
+
 ### Run the development server
 
-The project uses [Firebase Cloud Firestore](https://firebase.google.com/docs/firestore/) for product data. You will need to create a project and download the JSON configuration for the Admin SDK. You can do this in the [Firebase console](https://console.firebase.google.com/project/YOUR-PROJECT-NAME/settings/serviceaccounts/adminsdk) using the **Generate new private key** button. Save this to _src/data/firebase-admin-key.json_.
+The project uses [Firebase Cloud Firestore](https://firebase.google.com/docs/firestore/) for product data. You will need to create a project and download the JSON configuration for the Admin SDK. You can do this in the [Firebase console](https://console.firebase.google.com/project/YOUR-PROJECT-NAME/settings/serviceaccounts/adminsdk) using the **Generate new private key** button. Save this to `src/data/firebase-admin-key.json`.
```

Comment:
I don't feel strongly about this, but I tend to think URLs, paths and filenames should be italic, not code font.